web/source/keyboards/kmwkeyboards.ts

```diff
@@ -955,7 +974,7 @@ namespace com.keyman.keyboards {
           // Register any local keyboards immediately:
           // - must specify filename, keyboard name, language codes, region codes
           // - no request will be sent to cloud
-          var stub: KeyboardStub = <KeyboardStub> x[i];
+          let stub = x[i] as KeyboardStub;
```

Comment:
I know the original line suggested this with the type signature... but upon inspection, it's actually not an (internal) `KeyboardStub`.  It uses different nomenclature for the properties, so it's more of a "longhand" stub, if that makes sense.

Quite similar role, but the properties have to be mapped.

Reply:
I'll see if I can make use of the new `toStubs()` wrapper for this

Reply:
This does get mapped later on in l. 1084
```
let convertedStubs = KeyboardStub.toStubs(stub);
```

Reply:
> This does get mapped later on in l. 1084
> 
> ```
> let convertedStubs = KeyboardStub.toStubs(stub);
> ```

Then mark the type _there_, when it _is_ mapped.  As noted here, it gets _mapped_, and as is evident in the quoted line above, that result is instantly assigned to a new variable, not this one.  That new variable can receive this type annotation instead.
- Just to be extra explicit:  leaving `as KeyboardStub` on the original line will be considered grounds for rejection by itself.  Now that we know it's not valid, it'd be wrong to leave it.
- I guess it's kind of an "API stub" / "JSON stub"?  It's not currently a defined type, but commenting that may be appropriate if desired.

Also, after a double-check, it's not like `.toStubs` even needs the typing on the parameter; leaving it as an 'any' is totally fine here.

Reply:
Yeah, I'll just condense into a single line
```
let convertedStubs = KeyboardStub.toStubs(x[i]);
```